sdk/python/feast/infra/online_stores/snowflake.py

```diff
@@ -80,7 +83,7 @@ def online_write_batch(
                     "entity_feature_key",
                     "entity_key",
                     "feature_name",
-                    "value",
+                    f"{config.project}_value",
```

Comment:
Is the idea that multiple projects wold share a single table for the online store?

Reply:
@achals this can actually go back to value -- in general, if wanting to use snowflake as the entire offline,engine,online I need to introduce a protected keyword list ... basically you cant have a feature with the same column name in offline and online table when doing a direct UPSERT

I still need to add this

Reply:
But if using snowflake as offline+online store, there would still be two different schemas or tables, right? which should allow the same value name? 

(Partly I'm worried about the weird name, and partly that this change may break existing users of snowflake as the online store)

Reply:
@achals reverted to old schema